src/apps/search-result/facet-browser.test.ts

```diff
@@ -1,39 +1,118 @@
 const coverUrlPattern = /^https:\/\/res\.cloudinary\.com\/.*\.(jpg|jpeg|png)$/;
 
 describe("The Facet Browser", () => {
+  beforeEach(() => {
+    cy.interceptGraphql({
+      operationName: "searchFacet",
+      fixtureFilePath: "search-result/facet-browser/searchFacet"
+    });
+
+    cy.interceptGraphql({
+      operationName: "searchWithPagination",
+      fixtureFilePath: "search-result/facet-browser/searchWithPagination"
+    });
+
+    cy.interceptGraphql({
+      operationName: "intelligentFacets",
+      fixtureFilePath: "search-result/facet-browser/intelligentFacets.json"
+    });
+
+    cy.interceptRest({
+      aliasName: "Availability",
+      url: "**/availability/v3?recordid=**",
+      fixtureFilePath: "material/availability.json"
+    });
+
+    // Intercept all images from Cloudinary.
+    cy.intercept(
+      {
+        url: coverUrlPattern
+      },
+      {
+        fixture: "images/cover.jpg"
+      }
+    ).as("Harry Potter cover");
+
+    // Intercept covers.
+    cy.fixture("cover.json")
+      .then((result) => {
+        cy.intercept("GET", "**/covers**", result);
+      })
+      .as("Cover service");
+
+    // Intercept material list service.
+    cy.intercept("HEAD", "**/list/default/**", {
+      statusCode: 404,
+      body: {}
+    }).as("Material list service");
+
+    cy.visit("/iframe.html?id=apps-search-result--search-result");
+    cy.getBySel("facet-line-open-browser").click();
+  });
+
   it("renders all facets", () => {
-    cy.contains("Main languages");
-    cy.contains("Access types");
-    cy.contains("Children or adults");
-    cy.contains("Creators");
-    cy.contains("Fiction or nonfiction");
-    cy.contains("Genre and form");
-    cy.contains("Material types");
-    cy.contains("Subjects");
-    cy.contains("Work types");
+    cy.get('[aria-controls="facet-mainLanguages"]')
```

Comment:
Consider if this should use `cy` data attributes instead of `aria-controls` like we do elsewhere.